sdcm/nemesis/monkey/data_operations.py

```
@@ -0,0 +1,700 @@
+            clustering_keys = range(min_clustering_key + third_ck, max_clustering_key - third_ck)
+
+        if not clustering_keys:
+            clustering_keys = range(min_clustering_key, max_clustering_key)
+
+        self.runner.actions_log.info(
+            f"Delete same range in the few partitions in {ks_cf} table. Partitions count: {len(partitions_for_delete)}"
+        )
+        queries = []
+        for pkey in partitions_for_delete.keys():
+            queries.append(
+                f"delete from {ks_cf} where pk = {pkey} and ck >= {clustering_keys[0]} and ck <= {clustering_keys[-1]}"
```

Ruff (0.15.15)

[error] Possible SQL injection vector through string-based query construction

(S608)